### PR TITLE
Prepare azure

### PR DESCRIPTION
**Pull request by ubergesundheit**
This PR contains changes to the app required to run it on azure

### CODE REVIEW

File: sample_configs/values-gs.yaml [hunk 1]
```diff
@@ -50,7 +54,7 @@ s3_annotations: &s3-ann
 
 compactor: *s3-ann
 
-ingester: 
+ingester:
   <<: *s3-ann
```
Comment (webwurst) [inline sample_configs/values-gs.yaml:58]: Maybe it's fine to just copy-paste the annotation? Means duplication, but these yaml magic might also be a bit irritating for the end user. But just realizing this came in earlier. Also no strong opinion on this.

**Comment by webwurst** [comment]
Could parts of this also be interesting for upstream? Just to avoid us moving to far away of upstream :D

**Review by webwurst — APPROVED** [review]
Fine if it works for you. Only some minor comments.

File: sample_configs/values-gs.yaml (hunk 1 above)
Comment (ubergesundheit) [reply sample_configs/values-gs.yaml:58]: Yeah I see your concern. I'll write something as a comment to clarify this.

**Comment by ubergesundheit** [comment]
> Could parts of this also be interesting for upstream? Just to avoid us moving to far away of upstream :D

Yes, but the way upstream configures the different components is very generic. Would love having thoughts of @ced0ps and @piontec on this.

**Review by piontec — APPROVED** [review]
LGTM, but I agree that trying to submit it upstream without supporting all storage backends the same way might be hard. Still, maybe worth trying?

**Comment by ubergesundheit** [comment]
Yeah, upstreaming would make our lives easier.. I'll whip something up in grafana-helm-charts-upstream

**Pull request merged**